main.css

```diff
@@ -0,0 +1,500 @@
+  margin: 30px 0;
+}
+
+.partner-img {
+  width: 100px;
+  height: 100px;
+  margin: 0 15px;
+}
+
+footer {
+  display: flex;
+  justify-content: space-around;
+}
+
+.logo {
+  text-decoration-line: none;
+  font-family: 'COCOGOOSE', sans-serif;
+  font-weight: normal;
+  font-size: 20px;
+  color: #ec5242;
+  padding: 10px 10px;
+  background-color: #272a31;
+  text-align: center;
+  border-radius: 4px;
+  margin: 30px 10px;
+  line-height: 30px;
+}
+
+.copy-right {
+  font-family: 'COCOGOOSE', sans-serif;
+  font-weight: normal;
+  font-size: 14px;
+  margin: 50px 0;
+  text-align: center;
+}
+
+@media (max-width: 766px) {
+  nav .logo {
+    display: none;
+  }
+
+  .user-bar {
+    display: none;
+  }
+}
+
+@media (min-width: 768px) {
+  .hamb-icon {
+    display: none;
+  }
+
+  .user-bar {
+    width: 100%;
+    display: flex;
+    justify-content: flex-end;
+    background-color: #272a31;
+    height: 35px;
+  }
+
+  .user-bar-link {
+    text-decoration: none;
+    font-family: 'lato', sans-serif;
+    font-weight: normal;
+    font-size: 17px;
+    color: #fff;
+    padding: 0 20px;
+    cursor: pointer;
+  }
+
+  .user-bar-smedia {
+    width: 25px;
+    height: 25px;
+    cursor: pointer;
+    padding: 0 20px;
+  }
+
+  nav {
+    background-color: #fff;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    height: 95px;
+  }
+
+  .menu-list.close {
+    display: none;
+  }
+
+  .menu-links {
+    display: flex;
+    position: static;
+    box-shadow: none;
+    border-radius: 0;
+    flex-direction: row;
+    justify-content: space-around;
+    padding: 0;
+  }
+
+  .menu-link {
+    font-size: 13px;
+    color: #535353;
+    padding: 20px 15px;
+    margin: 0 15px;
+  }
+
+  .menu-link:hover {
+    color: #ec5242;
+  }
+
+  .menu-list.homepage {
+    display: none;
+  }
+
+  .main-title::before {
+    font-size: 35px;
+  }
+
+  .main-title {
+    font-size: 75px;
+    margin-left: 13%;
+  }
+
+  .headline-description {
+    font-size: 35px;
+    margin: 0 12%;
+  }
+
+  .event-date {
+    font-size: 40px;
+    margin-left: 12%;
+  }
+
+  .main-program-title,
+  .speakers-section-title,
+  .partner-title {
+    font-size: 26px;
+  }
+
+  .program-list {
+    flex-direction: row;
+    margin: 30px 1%;
+  }
+
+  .lectures {
+    flex-direction: column;
+    justify-content: space-around;
+    height: 200px;
+    max-width: 170px;
+  }
+
+  .lectures-title {
+    font-size: 18px;
+  }
+
+  .lectures-description {
+    font-size: 17px;
+    padding: 0 10px;
+  }
+
+  .main-program-btn {
+    margin-top: 40px;
+  }
+
+  .grid-container {
+    display: grid;
+    grid: repeat(2, 1fr)/ repeat(2, 1fr);
+    gap: 20px;
+  }
+
+  .speaker-img {
+    width: 150px;
+    height: 150px;
+  }
+
+  .speaker-name {
+    font-size: 20px;
+  }
+
+  .speaker-about {
+    font-size: 18px;
+  }
+
+  .talk-about {
+    font-size: 16px;
+  }
+
+  .partner-img {
+    width: 200px;
+    height: 200px;
+    margin: 0 30px;
+  }
```

Comment:
- [ ] Even though you can use whatever "partners" fit your project better, you should make the styling very similar to the original design. You could make the icons a bit smaller and add a [filter](https://developer.mozilla.org/en-US/docs/Web/CSS/filter-function/grayscale()) to make them match the behance design.

Behance design:
![image](https://user-images.githubusercontent.com/66967274/142279004-bd47755a-7343-4772-8579-60ea5b17482d.png)

Your design:
![image](https://user-images.githubusercontent.com/66967274/142279373-c7cd0e50-9a86-4ce6-a622-c392629ff23f.png)


Your design with smaller icons and greyscale filter:
![image](https://user-images.githubusercontent.com/66967274/142279336-18c5aef4-8613-4969-a7d3-a3f5f9f3c385.png)